1.6/run.sh

```diff
@@ -34,6 +34,11 @@ if [ -e /var/lib/jenkins/password ]; then
 fi
 rm -rf /tmp/war
 
+# Disable binding on HTTP port
+if [ -v JENKINS_OPTS ]; then
+  JENKINS_OPTS="--httpPort=-1 "
+fi
+
```

Comment:
I will, but want to get the other PR merged first to prevent conflicts ;-)

On Wed, Aug 26, 2015 at 4:55 PM, Ben Parees <notifications@github.com>
wrote:

> In 1.6/run.sh
> <https://github.com/openshift/jenkins/pull/21#discussion_r37991526>:
>
> > @@ -34,6 +34,11 @@ if [ -e /var/lib/jenkins/password ]; then
> >  fi
> >  rm -rf /tmp/war
> >
> > +# Disable binding on HTTP port
> > +if [ -v JENKINS_OPTS ]; then
> > +  JENKINS_OPTS="--httpPort=-1 "
> > +fi
> > +
>
> and remove this?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/jenkins/pull/21/files#r37991526>.
>



-- 
-----------------------------------------------
Michal Fojtik <mi@mifo.sk <mfojtik@mifo.sk>>
Red Hat OpenShift, Engineering